package.json

```diff
@@ -32,11 +32,14 @@
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static",
     "build": "lerna run --parallel build -- -- --env production",
     "build:storybook": "build-storybook -c storybook -o ./storybook/public -s ./storybook/static",
+    "setup": "git config core.hooksPath .githooks && npm run init",
```

Comment:
💯

Reply:
nit, I'd rename to `setup-hooks` or `install-hooks` or `setup:hooks` or `install:hooks`. And document that we have git hooks available.

Alternatively we can use [husky](https://github.com/typicode/husky) instead to manage git hooks. Very popular and widely used library for managing git hooks, should make maintaining them easier.

Reply:
If we add more complexity to the hooks, then would be a good time to explore using husky. Until then, the pre-push script is two lines and not worth a library.

I'm gonna add an `npm run setup:hooks`, and call that as part of `npm run setup`.